ctf_framework/templates/analytics/index.html

```diff
@@ -0,0 +1,30 @@
+{% extends "layouts/base.html" %}
+{% block content %}
+
+<div class="container">
+    <table id="dtVerticalScrollExample" scrollY="100px" class="table table-striped table-bordered table-sm" cellspacing="0" width="100%">
```

Comment:
We should rename the `id` to something relevant.

Reply:
will do